src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -144,6 +148,14 @@ class ReimbursementAccountPage extends React.Component {
         }
     }
 
+    fetchData() {
+        // We can specify a step to navigate to by using route params when the component mounts.
```

Comment:
This comment is out of date now as `fetchData()` is not only called on mount. Since we also call it when we are refetching the bank account it should be updated to explain why `stepToOpen` is initialized from the route params.